genesis/engine/entities/rigid_entity/rigid_entity.py

```diff
@@ -345,18 +345,34 @@ def _load_scene(self, morph, surface):
                     links_g_infos.insert(0, [])
                 assert len(links_g_infos_) == len(links_g_infos)
 
+                del links_g_infos_[0]
+                del links_j_infos_[0]
+                del l_infos_[0]
+                for l in l_infos_:
+                    l["parent_idx"] -= 1
+                    l["root_idx"] -= 1
+
+                # make timeconst 0 so it can be adjusted based on substep_dt
```

Comment:
Basically because it is a better default than mujoco ? If so, you should add this information in comment